amplify/backend/function/bridgeXYZMutation/src/handlers/updateExternalAccount.js

```diff
@@ -56,6 +41,26 @@ const updateExternalAccountHandler = async (
     account,
   );
 
+  const deletePromises = externalAccounts.map(async ({ id }) => {
+    const deleteAccountRes = await fetch(
+      `${apiUrl}/v0/customers/${bridgeCustomerId}/external_accounts/${input.id}`,
+      {
+        headers: {
+          'Api-Key': apiKey,
+        },
+        method: 'DELETE',
+      },
+    );
+
+    if (deleteAccountRes.status !== 200) {
+      console.log(
+        `Error deleting external account: ${await deleteAccountRes.text()}`,
+      );
+    }
+  });
```

Comment:
Would it make sense to extract also this into an utility function `deleteExternalAccount`?

```
const deleteExternalAccount = async (apiUrl, apiKey, bridgeCustomerId, id) => {
  const deleteAccountRes = await fetch(
    `${apiUrl}/v0/customers/${bridgeCustomerId}/external_accounts/${id}`,
    {
      headers: {
        'Api-Key': apiKey,
      },
      method: 'DELETE',
    },
  );

  if (deleteAccountRes.status !== 200) {
    console.log(
      `Error deleting external account: ${await deleteAccountRes.text()}`,
    );
  }
}
```

And then use it here

```suggestion
  const deletePromises = externalAccounts.map(({ id }) => 
     deleteExternalAccount(apiUrl, apiKey, bridgeCustomerId, id)
  );
```